packages/lambda-powertools-pattern-basic/index.js

```diff
@@ -12,7 +12,24 @@ if (!process.env.DATADOG_PREFIX) {
   process.env.DATADOG_PREFIX = FUNCTION_NAME + '.'
 }
 
-process.env.DATADOG_TAGS = `awsRegion:${AWS_REGION},functionName:${FUNCTION_NAME},functionVersion:${FUNCTION_VERSION},environment:${ENV}`
+const supplementCsv = (existing, toAdd) => {
+  // existing will be in the format '<key1>:<value1>,<key2>:<value2>'
+  // Map requires as [[key1, value1], [key2, value2]]
+  const existingNormalised = existing.split(',').map(pair => pair.split(':'))
```

Comment:
```suggestion
  const existingNormalised = existing && existing.split(',').map(pair => pair.split(':')) || []
```

Reply:
@gsingh1  - I've reworked this change considerably, which should not only address this but also make using supplementing the csv data easier to use if we pull it into the other patterns